Pomodoro.Tests/DataAccessTests/FrequencyRepositoryTests.cs

```diff
@@ -0,0 +1,197 @@
+﻿// <copyright file="FrequencyRepositoryTests.cs" company="PomodoroGroup_GL_BaseCamp">
+// Copyright (c) PomodoroGroup_GL_BaseCamp. All rights reserved.
+// </copyright>
+
+using Pomodoro.DataAccess.EF;
+using Pomodoro.DataAccess.Entities;
+using Pomodoro.DataAccess.Repositories.Realizations;
+using Pomodoro.Tests.EqualityComparers;
+
+namespace Pomodoro.Tests.DataAccessTests
+{
+    /// <summary>
+    /// Frequency repository test class.
+    /// </summary>
+    public class FrequencyRepositoryTests
+    {
+        /// <summary>
+        /// Adds frequency to database.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> object that represents an asynchronous operation.</returns>
+        [Fact]
+        public async Task AddAsync_AddsFrequencyToDatabase()
+        {
+            // arrange
+            using var context = new AppDbContext(UnitTestHelper.GetUnitTestDbOptions());
+            var frequencyRepository = new FrequencyRepository(context);
+            var frequency = new Frequency
+            {
+                Id = new Guid(2, 8, 3, new byte[] { 0, 1, 2, 3, 4, 5, 6, 7 }),
+                FrequencyTypeId = new Guid(7, 2, 3, new byte[] { 0, 1, 2, 3, 4, 5, 6, 7 }),
+                Every = 2,
+            };
+
+            // act
+            await frequencyRepository.AddAsync(frequency);
+            await context.SaveChangesAsync();
+
+            // assert
+            Assert.Equal(8, context.Frequencies.Count());
```

Comment:
Maybe it's not a good idea to use the constant 8 here. It's not clear why the Count should be 8 because you added only 1 record to the DB. What about something like this:

`int expectedCount = context.Frequencies.Count() + 1;`
`// act ...`
`// assert`
`Assert.Equal(expectedCount, context.Frequencies.Count());`

Reply:
You are right. Done)